lib/chef/provider/package/homebrew.rb

```diff
@@ -83,9 +87,40 @@ def brew(*args)
         # information, but that is not any more robust than using the
         # command-line interface that returns the same thing.
         #
-        # https://github.com/Homebrew/homebrew/wiki/Querying-Brew
+        # https://docs.brew.sh/Querying-Brew
+        #
+        # @returns [Hash] a hash of package information where the key is the package name
         def brew_info
-          @brew_info ||= Chef::JSONCompat.from_json(brew("info", "--json=v1", new_resource.package_name)).first
+          @brew_info ||= begin
+            command_array = ["info", "--json=v1"].concat Array(new_resource.package_name)
```

Comment:
`Array(new_resource.package_name)` should just be `package_name_array`

Reply:
Done